ext/openssl/ossl_pkey.c

```diff
@@ -816,6 +881,33 @@ ossl_pkey_private_to_pem(int argc, VALUE *argv, VALUE self)
     return do_pkcs8_export(argc, argv, self, 0);
 }
 
+/*
+ *  call-seq:
+ *     key.private_to_raw   => string
+ *
+ *  See the OpenSSL documentation for EVP_PKEY_get_raw_private_key()
+ */
+
+#ifdef HAVE_EVP_PKEY_NEW_RAW_PRIVATE_KEY
+static VALUE ossl_pkey_private_to_raw(VALUE self)
+{
+    EVP_PKEY *pkey;
+    VALUE str;
+    size_t len;
+
+    GetPKey(self, pkey);
+    EVP_PKEY_get_raw_private_key(pkey, NULL, &len);
```

Comment:
This probably needs an error check.

Reply:
Fixed @ [6628df1](https://github.com/ruby/openssl/pull/646/commits/6628df1adfcad9100095c26b17c2fa3d4e043740).
Maybe we could have a different error message for this call and the next call, but it could be difficult to tell what exactly caused the error on each call so I think leaving as it is now would be reasonable.